### DISCUSSION TITLE
VS 17.4 Preview 2.1 .jar files showing in project folder

### DISCUSSION THREAD

**Discussion by tdeborde2** [dotnet/maui#10242] [Q&A] [answered]
Installed VS 17.4 Preview 2.1 on MacOS and I am seeing a bunch of jar files in the project folder.

The files can be removed from the project without issue but return when the solution is closed and reopened. I am seeing the same behavior with a new project created from the MAUI App template. Has anyone ran into this and is there a way to resolve it?

This does not seem to cause any problems, just clutters up the solution explorer. 

<img width="247" alt="Screen Shot 2022-09-21 at 9 10 10 AM" src="https://user-images.githubusercontent.com/8270104/191542624-7fb628c3-f816-4674-b6e1-a3a2b79707e4.png">

**Comment by nicjay** [comment]
Having the same issue. It seems related to net6.0-android target in `<TargetFrameworks>` in the .csproj. When it's removed the .jar files disappear, although that's obviously not a solution if you're targeting Android.

**Comment by magnusguvenal** [comment]
Would also like to have a solution for this, don't like it this way to much clutter

**Reply by EdwardMcFlurry** [reply]
Is the Team even aware of this???

**Comment by julianadormon** [comment]
For me as well.

**Comment by golovko** [comment]
The same

**Comment by tdeborde2** [comment, marked as answer]
Updated to VS 2022 for Mac 17.4 preview 3.0 and the issue appears to be resolved.

**Reply by golovko** [reply]
Solved! Thanks.
